src/PluginRTCDataChannel.swift

```diff
@@ -245,18 +223,9 @@ class PluginRTCDataChannel : NSObject, RTCDataChannelDelegate {
 		}
 	}
 
-	func channel(_ channel: RTCDataChannel!, didChangeBufferedAmount amount: UInt) {
-		NSLog("PluginRTCDataChannel | didChangeBufferedAmount %d", amount)
-
-		self.eventListener!([
-			"type": "bufferedamount",
-			"bufferedAmount": amount
-			])
-
-	}
```

Comment:
Why have you removed this? `bufferedAmount`is something dynamic that changes, not a static value that can just be read at the beginning.